FrontEnd/src/Components/ModalForm.jsx

```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
-                      color="#333333"
-                      bg="white"
-                      borderRadius="4px"
-                    />
-                    <Text fontSize="14px" color="#B3B3B3" textAlign="right">
-                      0/500
-                    </Text>
-                  </FormControl>
-                  <Flex
-                    justifyContent="space-between"
-                    alignItems="center"
-                    mt="24px"
-                  >
-                    <Box>
-                      <Text fontSize="14px" fontStyle="italic" color="#B3B3B3">
-                        *All the Fields are Mandatory to fill
-                      </Text>
-                    </Box>
-                    <Box justifyContent="space-between">
-                      <Button
-                        bg="#E5E3FF"
-                        color="#6C63FF"
-                        borderRadius="3px"
-                        onClick={handleCloseModal}
-                        mr="30px"
-                        _hover={{}}
-                        disabled={
-                          !formValues.promptType ||
-                          !formValues.promptName ||
-                          !formValues.promptDescription
-                        }
-                      >
-                        Cancel
-                      </Button>
-                      <Button
-                        bg="#6C63FF"
-                        color="white"
-                        borderRadius="3px"
-                        onClick={() => setPage(2)}
-                        _hover={{}}
-                        disabled={
-                          !formValues.promptType ||
-                          !formValues.promptName ||
-                          !formValues.promptDescription
-                        }
-                      >
-                        Next
-                      </Button>
-                    </Box>
-                  </Flex>
-                </Box>
+                <Form1
+                  formValues={formValues}
+                  handleInputChange={handleInputChange}
+                  handleCloseModal={handleCloseModal}
+                  nextPage={handleNextPage}
+                />
               )}
               {page === 2 && (
-                <Box>
-                  <FormControl id="message">
-                    <FormLabel>Message</FormLabel>
-                    <Textarea
-                      name="message"
-                      value={formValues.message}
-                      onChange={handleInputChange}
-                    />
-                  </FormControl>
-                  <Stack direction="row" justifyContent="space-between">
-                    <Button onClick={() => setPage(1)}>Back</Button>
-                    <Button colorScheme="blue" onClick={handleSubmit}>
-                      Submit
-                    </Button>
-                  </Stack>
-                </Box>
+                <Form2
```

Comment:
Please use descriptive names for components

Reply:
Got it, will make changes!